Makefile

```diff
@@ -175,13 +177,9 @@ ${GINKGO}: ${BUILD_BIN_PATH}
 ${MOCKGEN}:
 	$(call go-build,mockgen,./vendor/github.com/golang/mock/mockgen)
 
-vendor: .install.vndr
-	$(GOPATH)/bin/vndr \
-		-whitelist github.com/onsi/ginkgo \
-		-whitelist github.com/golang/mock \
-		-whitelist github.com/cpuguy83/go-md2man \
-		-whitelist github.com/russross/blackfriday \
-		${PKG}
+vendor:
+	$(GO) mod tidy
+	$(GO) mod vendor
```

Comment:
`$(GO) mod verify` to be ultra sure nobody tricked us :)

Reply:
Alright, I added it :+1: